__tests__/app.test.js

```diff
@@ -197,4 +197,76 @@ describe("NC News API testing", () => {
         });
     });
   });
+  describe("POST /ap/articles/:article_id/comments - will add a comment for the requested aticle", () => {
+    test("will respond with a 201 status code and the posted comment", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "I hope you get this comment",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          const addedComment = response.body.comment;
+          expect(typeof addedComment.comment_id).toBe("number");
+          expect(addedComment.body).toBe("I hope you get this comment");
+          expect(addedComment.article_id).toBe(1);
+          expect(addedComment.author).toBe("butter_bridge");
+          expect(typeof addedComment.votes).toBe("number");
+          expect(typeof addedComment.created_at).toBe("string");
+        });
+    });
+    test("will respond with 400 status code and an error message when the username doesn't exist", () => {
+      const newComment = {
+        username: "banana",
+        body: "This should not get posted on the article",
+      };
+      return request(app)
+        .post("/api/articles/7/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Username does not exist!");
+        });
```

Comment:
Nice this is a test, but it should be 404 instead of a 400 error. This is because banana could be a username but it simply isnt one right now